src/networkremote/networkremote.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Strawberry Music Player
+ * Copyright 2025, Leopold List <leo@zudiewiener.com>
+ *
+ * Strawberry is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Strawberry is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Strawberry.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include <QThread>
+#include "networkremote/networkremote.h"
+#include "core/application.h"
+#include "core/logging.h"
+
+NetworkRemote* NetworkRemote::sInstance_ = nullptr;
+
+NetworkRemote::NetworkRemote(Application* app, QObject *parent)
+    : QObject(parent),
+      app_(app),
+      enabled_(false),
+      local_only_(false),
+      remote_port_(5050),
+      server_(nullptr),
+      settings_(new NetworkRemoteSettings()) {
+  setObjectName("NetworkRemote");
+  sInstance_ = this;
+}
+
+NetworkRemote::~NetworkRemote() {
+  stopTcpServer();
+}
+
+void NetworkRemote::Init() {
+  LoadSettings();
+  if (enabled_) {
+    startTcpServer();
+  }
+  else {
+    stopTcpServer();
+  }
+  qLog(Debug) << "NetworkRemote Init() ";
```

Comment:
Is there a reason why this comment is not at the start of the function?

Reply:
Moved